tests/test_make.py

```diff
@@ -246,26 +246,30 @@ class C(object):
         assert sentinel == getattr(C, method_name)
 
     @pytest.mark.skipif(not PY3, reason="__qualname__ is PY3-only.")
-    def test_repr_qualname(self):
+    @pytest.mark.parametrize('slots_outer', [True, False])
```

Comment:
It's not the same though, doing it this way will produce a cartesian product (i.e. test all combinations). Want me to put in all four?